build.gradle

```diff
@@ -18,6 +18,13 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven {
```

Comment:
You won't be able to build with this because the CI is not adding in the build variables for this repo.  This would require additional CI changes.